main.🍇

```diff
@@ -30,6 +30,11 @@
     🔤t-rex🔤 ➡️ 🆕😶 🔤🦖🔤 🍿 🔤animal🔤 🍆 ❗️
     🔤slightly-smiling-face🔤 ➡️ 🆕😶 🔤🙂🔤 🍿 🔤face🔤 🔤smile🔤 🍆 ❗️
     🔤smirking-face🔤 ➡️ 🆕😶 🔤😏🔤 🍿 🔤face🔤 🔤smug🔤 🔤joke🔤 🍆 ❗️
+    🔤fire🔤 ➡️ 🆕😶 🔤🔥🔤 🍿 🔤fire🔤 🔤hot🔤 🔤heat🔤 🍆 ❗️
+    🔤snowflake🔤 ➡️ 🆕😶 🔤️❄️🔤 🍿 🔤ice🔤 🔤winter🔤 🔤snow🔤 🍆 ❗️
```

Comment:
```suggestion
    🔤snowflake🔤 ➡️ 🆕😶 🔤️❄️🔤 🍿 🔤ice🔤 🔤winter🔤 🔤snow🔤 🔤cold🔤 🍆 ❗️
```
Since :fire: has `hot`, I think it makes sense for :snowflake: to have `cold`.